src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -505,7 +505,10 @@ private void CleanupDatabaseRelatedValues(TransactionOperationContext context, T
             }
 
             DeleteTreeByPrefix(context, UpdateValueForDatabaseCommand.GetStorageKey(databaseName, null), Identities);
-            DeleteTreeByPrefix(context, databaseName + "/", CompareExchange, RootObjectType.Table);
+            using (Slice.From(context.Allocator, databaseName, out var databaseSlice))
+            {
+                context.Transaction.InnerTransaction.OpenTable(CompareExchangeSchema, CompareExchange).DeleteByPrimaryKeyPrefix(databaseSlice);
+            }
```

Comment:
What is we have a another database with the same prefix?
We need to have here a separator.

Reply:
https://github.com/ravendb/ravendb/pull/4878